File "tests/record_wrong_accessor.meja", line 9, characters 8-9:
Error: Incompatable types X.t(_) and t(field, field, field):
       Cannot unify X.t('a) and t(field, field, field)
